qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -421,10 +421,9 @@ def closeEvent(self, event):
                 event.ignore()
                 return
 
+        self.writeSettings(CONF)  # write current window information to global settings object
```

Comment:
Is it still necessary to have this call here? It means we save settings even in the case where a user clicks cancel on stops the close event.